azure_pipelines/templates/tests-with-conf-file.yml

```diff
@@ -39,7 +39,7 @@ steps:
     inputs:
       targetType: 'inline'
       script: |
-        mv azure_pipelines/scripts/conf.json MSAL/test/automation/conf.json
+        cp azure_pipelines/scripts/conf.json MSAL/test/automation/conf.json
```

Comment:
is this change required? Why are we copying the conf.json file instead of moving it?

Reply:
It's not, I thought the conf file was getting downloaded on the main pipeline but it was actually not. Removed